embulk-output-jdbc/src/main/java/org/embulk/output/jdbc/AbstractJdbcOutputPlugin.java

```diff
@@ -760,6 +761,22 @@ private void dropTables() throws SQLException
         }
     }
 
+    @VisibleForTesting
+    static int calculateSuffixLength(int taskCount)
+    {
+        assert(taskCount >= 0);
+        return Math.max(1, String.valueOf(taskCount - 1).length());
+    }
+
+    protected String generateIntermediateTableNameFormat(String baseTableName, JdbcOutputConnection con,
```

Comment:
I avoided writing tests for this method because I would have to make changes to parts that are not related to this PR. Since the internal implementation is time-dependent, it is impossible to write tests without first separating the dependency.